Examples/Tests/openbc_poisson_solver/inputs_3d

```diff
@@ -8,17 +8,17 @@ amr.n_cell = 128 128 128
 amr.max_level = 0
 amr.max_grid_size = 128
 geometry.dims = 3
-geometry.prob_lo = -5*sigmax -5*sigmay -5*sigmaz
-geometry.prob_hi = +5*sigmax +5*sigmay +5*sigmaz
+geometry.prob_lo = -4*sigmax -4*sigmay -4*sigmaz
+geometry.prob_hi = +4*sigmax +4*sigmay +4*sigmaz
```

Comment:
Here the resolution was effectively increased by reduced the physical size of the box.